rgw/v2/tests/s3_swift/test_swift_basic_ops.py

```diff
@@ -124,12 +125,12 @@ def test_exec(config):
     log.info(type(ceph_conf))
     rgw_service = RGWService()
     # preparing data
-    user_names = ["tuffy", "scooby", "max"]
+    user_name = names.get_first_name()
```

Comment:
Could you please append 1 or 2 random character to the user name since sometimes `get_first_name`returns same name

Reply:
@vivekanandan-k-rh I tried running `get_first_name()` several times on my local computer and I can see repeated names.
If we are sure that it is not taking duplicate names, we can go with the same code

Reply:
@udaysk23 , Following was the result for 10 iterations:

>>> for i in range(10):
...   print(names.get_first_name())
... 
Jerry
Rosa
Lawrence
Mary
Paul
David
Cody
Nicole
Jean
George

Reply:
Done @udaysk23 . Added random character with first name